ui/js/dfv/src/hooks/useBlockEditor.js

```diff
@@ -0,0 +1,77 @@
+
+const useBlockEditor = () => {
+	const editorSelect = wp.data?.select( 'core/editor' );
+	const editorDispatch = wp.data?.dispatch( 'core/editor' );
+	const notices = wp.data?.dispatch( 'core/notices' );
+
+	// @todo Use hook instead of savePost override once stable.
+	if ( ! window.PodsBlockEditor && editorDispatch && editorDispatch.hasOwnProperty( 'savePost' ) ) {
+		// First init.
+		window.PodsBlockEditor = {
```

Comment:
Optionally make this a class with private props and getters/setters. Maybe even an external tool like the API instead of a React hook.

Reply:
Lets keep it like this since we should definitely change this code once a hook is available.
The added todo will inform any dev checking this code in that it's a temp solution.